src/goto-symex/field_sensitivity.cpp

```diff
@@ -181,6 +181,28 @@ void field_sensitivityt::field_assignments(
   run_apply = run_apply_bak;
 }
 
+void field_sensitivityt::post_process_assignment(
+  goto_symex_statet &state,
+  const ssa_exprt &lhs,
+  const namespacet &ns)
+{
+  if(is_divisible(lhs))
+  {
+    // Split composite symbol lhs into its components:
+    field_assignments(state, lhs);
+
+    // Erase the composite symbol from our working state
+    // (otherwise keeping both the composite and field representations risks
+    // the two becoming out of sync).
+    // Note we can't use code_deadt, which would actually try to kill the
+    // components, which we *do* want to keep!
+    state.propagation.erase(lhs.get_l1_object_identifier());
+
+    exprt unknown_expr(ID_unknown, lhs.type());
```

Comment:
{ } and we should have an `unknown_exprt` type

Reply:
This is now gone.